packages/react-ui-map/src/Mapbox/MapLayer.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+export default class MapLayer extends Component {
+  static propTypes = {
+    layers: PropTypes.array
+  }
+
+  static contextTypes = {
+    map: PropTypes.object
+  }
+
+  componentWillMount() {
```

Comment:
Can you place the code from `componentWillMount` ([from docs](https://facebook.github.io/react/docs/react-component.html#componentwillmount)) in the `constructor` instead?